geopandas/sindex.py

```diff
@@ -206,6 +206,45 @@ def intersection(self, coordinates):
         """
         raise NotImplementedError
 
+    def nearest_all(self, geometry, max_distance=None, return_distance=False):
+        """Return the nearest geometry in the tree for each input geometry in
+        ``geometry``.
+
+        Note: ``nearest_all`` currently only works with PyGEOS >= 0.10.
```

Comment:
```suggestion
        .. note::
            ``nearest_all`` currently only works with PyGEOS >= 0.10.
```

This will make an admonition in docs.